src/taggle.js

```diff
@@ -268,17 +270,20 @@
                 return;
             }
 
+            if (!settings.onBeforeTagAdd(e, val)) {
```

Comment:
If your use case is validation, couldn't you use the [`allowedTags `](https://github.com/okcoker/taggle.js/pull/19/files#diff-e113ff04b2bf8f68fa28e11be484f9deR55) option? There's a [return statement](https://github.com/okcoker/taggle.js/pull/19/files#diff-e113ff04b2bf8f68fa28e11be484f9deR269) above this one.